test/src/test/kotlin/graphql/nadel/tests/next/NadelIntegrationTest.kt

```diff
@@ -126,11 +126,15 @@ abstract class NadelIntegrationTest(
         // Maybe this won't hold out longer term, but e.g. it's ok for the deferred errors to add a path
         assertJsonEquals(
```

Comment:
Want to ignore "locations" field in errors when comparing defer vs non-defer queries because the line/column of the error would be different as the two queries look slightly different (one has `@defer` directive stripped)